docker-compose-develop.yml

```diff
@@ -39,11 +43,33 @@ services:
       AREAS_API_URL: http://127.0.0.1:9000
       GEOSTORE_API_URL: http://127.0.0.1:9000
       API_VERSION: v1
+      MONGODB_HOST: mongo
+      MONGODB_PORT: 27017
+      DB_SECRET: '{ "username": "admin", "password": "password" }'
+      DB_DATABASE: fw_api_db
     volumes:
```

Comment:
can we add
```yaml
networks:
      - gfw-fw-network
      - gfw-network
```

so the tests can connect to the DB.

can you include that in the doc too?